content/6_Advanced/SPneg.mdx

```diff
@@ -362,6 +362,57 @@ public class CyclesFinding {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+INF = 10**18
```

Comment:
you know

Reply:
I already mentioned it above that using float('inf') will result in WA for 3 TCs.

Reply:
so put it in a comment, the readers don't read your pr descriptions

Reply:
done